### ISSUE TITLE
CH 36 | Update VA File Number Component

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68001]
## Issue Description
_CAIA #63707 noted that the file VA file number field is confusing, so it needs to be updated to follow design system guidance._

## Details
This is an issue with a field label on [Step 1 of 2: Applicant Information](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/claimant-information).

![Screenshot 2023-08-15 at 4 18 13 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/c259f8e0-a4ca-4186-af9e-40a62c3d4b75)

"Your VA file number (*If known)" doesn't follow [design system guidelines](https://design.va.gov/patterns/ask-users-for/social-security-number#usage). And it's confusing for users to see the "required" visual pattern - red text with an asterisk - next to a field that isn't required.

### Staging or Production Link
- [Staging Link](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-25-8832/introduction)

### Prototype Link
- [Mural User Flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691434304975/3f0c405c820c184d21393b3bbd76324a5ee7c677?sender=u0e8ac1d6d7681bb7e1b80558)

### Test Users
- vets.gov.user+24@gmail.com
- [staging test user repository](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv)

## Reproduction steps
1. Go to the [unauthenticated form](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/claimant-information)
2. Scroll down to "Your VA file number"

---
## Tasks
- [ ] Change field name to: VA file number if known
- [ ] Add helper text below file number field name to align with the design system guidelines for [VA file number](https://design.va.gov/patterns/ask-users-for/social-security-number#usage)
   - You must enter either a VA file number or Social Security number

## Acceptance Criteria
- [ ] Form field title updated
- [ ] Component aligns with design system guidelines
- [ ] Fiorella or Julie sign off on design in staging
---

**Comment by Midge-dev** [comment]
Commented here https://github.com/department-of-veterans-affairs/va.gov-team/issues/63707

**Comment by steele-lm** [comment]
@zack I confirmed with Julie that this ticket should be actionable without additional design support.
